client/src/components/v2/OverlaySelection.stories.tsx

```diff
@@ -10,28 +10,28 @@ export default {
 
 const Template = addArgs<OverlaySelectionProps>((props) => <OverlaySelection {...props} />);
 
-const setActiveOverlay = action('set');
+const baseProps = {
+    device: 'primary' as 'primary', // For literal type, can use Device when that gets merged
```

Comment:
Next time drop a `FIXME:` for this kind of comment so it highlights in editors.
See https://marketplace.visualstudio.com/items?itemName=wayou.vscode-todo-highlight